target/linux/kirkwood/base-files/etc/uci-defaults/04_storafan

```diff
@@ -0,0 +1,22 @@
+#
+# Copyright (C) 2021 OpenWrt.org
+#
+
+. /lib/functions.sh
+
+board=$(board_name)
+
+case "$board" in
+netgear,stora)
```

Comment:
If it can be extended to accommodate other devices as well, then why use the board-specific filename?

Reply:
> If it can be extended to accommodate other devices as well, then why use the board-specific filename?

mvebu has a similar script (mvebu/cortexa9/base-files/etc/uci-defaults/04_mambafan). Some unification might be done with these, but I'm not in a position to change mvebu.

Reply:
Note: The Netgear WNDR4700 uses a tc654 too. However instead of a script, it uses thermal-zones in the device-tree to control the fan. (With a special boot script /etc/init.d/hwmon_fancontrol to put the chip into pwm mode)